cluster/terraform_aks_cluster/config/production.tfvars.json

```diff
@@ -6,7 +6,19 @@
   "node_pools": {
     "applications": {
       "min_count": 3,
-      "max_count": 20
+      "max_count": 10,
+      "vm_size": "Standard_E4ads_v5",
```

Comment:
This may recreate the pool and disrupt the apps. Can we add it later?

Reply:
Good point!